applications/CompressiblePotentialFlowApplication/python_scripts/forces_elements_to_nodes_process.py

```diff
@@ -0,0 +1,47 @@
+import KratosMultiphysics
+from KratosMultiphysics.CompressiblePotentialFlowApplication.compute_lift_process import ComputeLiftProcess
+
+def Factory(settings, Model):
+    if(not isinstance(settings, KratosMultiphysics.Parameters)):
+        raise Exception(
+            "expected input shall be a Parameters object, encapsulating a json string")
+    return ForcesElementsToNodesProcess(Model, settings["Parameters"])
+
+# all the processes python processes should be derived from "python_process"
+
+class ForcesElementsToNodesProcess(ComputeLiftProcess):
+    def __init__(self, Model, settings):
+        super(ForcesElementsToNodesProcess, self).__init__(Model, settings)
+
+    def ExecuteFinalizeSolutionStep(self):
```

Comment:
Could you put everything into a different function so that we can call it from the FSI loop? I mean something like:

```python
def ExecuteFinalizeSolutionStep(self):
    self.ComputeForcesOnNodes()
def ComputeForcesOnNodes(self) :
```

Reply:
Thank You @inigolcanalejo !
About the Test, Lines 56-64 and Lines 119-131 in the naca0012_small_parameters.json the required lines are written and when running potential_flow_test_factory.py, the results of the new class will appear. I will add a print line saying that the Forces has been transfered in the next commit.
As for the function I will add it as well to the next commit.